addresses/frontend-mainnet.addresses.json

```diff
@@ -109,6 +109,8 @@
         "0x7320d680Ca9BCE8048a286f00A79A2c9f8DCD7b3",
         "0x49e9e169f0B661Ea0A883f490564F4CC275123Ed",
         "0x4bd6D86dEBdB9F5413e631Ad386c4427DC9D01B2",
-        "0xCF354603A9AEbD2Ff9f33E1B04246d8Ea204ae95"
+        "0xCF354603A9AEbD2Ff9f33E1B04246d8Ea204ae95",
+        "0x7E4FEE0C14673dB4F1526fdbeA53819e5808a9b5",
+        "0x28b0379D98fb80Da460C190c95f97C74302214B1"
```

Comment:
Need to include the bb-a-usd address in the top section of this file.

Reply:
Thanks Danny! Pushed another commit which add the address. Cheers!